src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -13619,7 +13631,7 @@ void Test23(CL0.CL1 c, Action x23)
             c.VerifyDiagnostics(expected);
         }
 
-        [Fact]
+        [Fact(Skip = "NullableOptOut does not control warnings")]
```

Comment:
What's the plan for those skipped tests? Should they be updated with the diagnostics that are produced?